common/src/main/java/com/wynntils/features/user/WorldWaypointDistanceFeature.java

```diff
@@ -84,7 +88,10 @@ public void onRenderLevelPost(RenderLevelEvent.Post event) {
 
     @SubscribeEvent
     public void onRenderGuiPost(RenderEvent.Post event) {
-        if (CompassModel.getCompassLocation().isEmpty() || normalizedDeviceCoordinates == null || isOnScreen()) return;
+        if (CompassModel.getCompassLocation().isEmpty()
+                || normalizedDeviceCoordinates == null
+                || isOnScreen()
+                || maxWaypointTextDistance < distance) return;
```

Comment:
Would like it so if 0 would mean display every time

Reply:
Added this in the most recent commit